src/main.cc

```diff
@@ -1,3 +1,10 @@
+/**
+ * See ../include/_nullptr (Reference to Scott Meyer)
+ */
+#if _MSC_VER < 1600 //MSVC version < 8
+     #include "../include/_nullptr.h"
```

Comment:
I switched this over to `src/` locally. Most of my changes were kept local today and pretty hurridly because I was testing compatibility with my project. I'll push most of the stuff I have on my side later. I apologize for the intermediate sloppiness